src/tests.rs

```diff
@@ -28,39 +32,122 @@ fn test_sums_match() {
             .parse::<GeoJson>()
             .unwrap();
 
-        let output_sum = sum_trips_output(&output);
+        for (column, input_sum) in &input_sums {
+            let input_sum = *input_sum;
+            let output_sum = sum_trips_output(&output, column);
+            let epsilon = 1e-6;
+            assert!(
+                (input_sum - output_sum).abs() < epsilon,
+                "Number of {} trips in input {} and jittered output {} don't match for max_per_od = {}",
+                column,
+                input_sum,
+                output_sum,
+                max_per_od
+            );
+        }
+    }
+}
+
+#[test]
```

Comment:
The unit test code is very verbose and awkward; I'm not happy with it at all, will keep iterating.